data/xml/2023.dstc.xml

```diff
@@ -18,6 +18,7 @@
       <month>September</month>
       <year>2023</year>
       <venue>dstc</venue>
+      <venue>ws</venue>
```

Comment:
@mjpost So we have two tags -- `dstc` (workshop) venue tag and `ws` tag?

Reply:
Yes. Every volume has a tag for its venue. Then, if it is a workshop, we also add ws. No volume can have *just* ws (except old ones that we haven't fixed yet). 

This was tag replaces back linking from the YYYY.ws.xml file.

Reply:
Also this needs to be added to each workshop in the PR.